task/rpms-signature-scan/0.1/rpms-signature-scan.yaml

```diff
@@ -80,7 +80,7 @@ spec:
           --fail-unsigned "${FAIL_UNSIGNED}" \
           --workdir "${WORKDIR}" \
     - name: output-results
-      image: quay.io/konflux-ci/konflux-test:v1.4.28@sha256:4a5423e125fc28db800421422d9933290dc4b62a22401d74cd3348c03107a5d9
+      image: quay.io/konflux-ci/konflux-test:v1.4.20@sha256:f9db697d8a45870b862252de61b3c29d9d6f79272ef8bf61ecb645f8bca27705
```

Comment:
I'm confused - is this a regression?

Reply:
Hmmm... The task was modified directly in build-definitions, and now the task that updates it automatically is reverting the change. This task is deprecated so we don't update it upstream anymore.